docs-source/content/userguide/cicd/choose-an-approach.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Choose an approach"
```

Comment:
Reader view comment: Use "Choosing an approach"?

Reply:
We've been using imperative verbs versus gerunds, therefore, "Choose an approach" is correct.

Reply:
aha!!